### ISSUE TITLE
update the wrapper so it works with newer versions of the beta

### ISSUE THREAD

**Issue by Goldensteev** [zackphillips/micro-manager-python3#1]
Hey, 

I don't know how easily it can be done, but i'm trying to have the MMCorePy library work with the newer versions of the micromanager beta or gamma branch, unfortunately, your compilation is only compatible with the build up to June 2016, and some of my devices are only compatible with newer beta/gamma branches. I tried doing it myself but i can't get it to work! Could you if it's not to hard to do since you've done it once, recompile the library with newer micromanager branches?  

EDIT : it's not really urgent or super important, so if it takes little to no time to do i would love if you do, otherwise don't bother! 

Thanks in advance!

**Comment by zfphil** [comment]
I'm actually on internship for the next 3 weeks so it's tough to find time at the moment - after the end of August, I will certainly update the binaries, and ideally write up a guide for how I did this. In the meantime, I used this guide to set-up the compilation if you're interested in trying yourself:

https://micro-manager.org/wiki/Building_MM_on_Windows

I did this 1.5 years ago now so I'm short on details, but I recall using the latest VS Community edition with the Win7 SDK, and changing the MMCorePy solution properties to point to my anaconda python installation. Hope this is helpful in the meantime!

**Comment by Goldensteev** [comment]
Thanks for the response, 
so after a bit of looking around I've actually managed to compile my drivers with the version you compiled the MMCorePy. It's not ideal as i'm not really up to date with the nightlies but in any case it works great for now! I had tried to do it myself but i was having too many troubles (only regarding the MMCorePy compilation). Anyway it seems as though the MM team will be moving to 3.6 soon anyway, so if it's still not official by the end of your internship, the new version you compile will be welcome!

Thanks again!
